experimental/sdg-tracker/src/App.tsx

```diff
@@ -45,7 +45,7 @@ const InitializeStore: React.FC = () => {
   const initializeAppState = useStoreActions((a) => a.initializeAppState);
   useEffect(() => {
     initializeAppState();
-  }, []);
+  }, [initializeAppState]);
```

Comment:
should really test out these dependency... I have noticed that these are indeed issues when we initially develop, but they may introduce breakage, and need property bigger fix.

Reply:
Ok, reverted updating the dependency arrays for now.